src/containers/Equipe/Equipe.js

```diff
@@ -1,20 +1,50 @@
 import React from 'react'
-import { Container } from 'semantic-ui-react'
+import { Container, Card, Image } from 'semantic-ui-react'
 import { FormattedMessage } from 'react-intl'
 
-const Equipe = () =>
-  <div className='equipe'>
-    <Container className='container'>
-      <h2><FormattedMessage id='equipe.title' /></h2>
-      <div class='ui grid'>
-        <div class='six wide column'>
-          <img src='img/test.jpg' alt="" />
-        </div>
-        <div class='ten wide column'>
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ullamcorper commodo nisi vel commodo. Suspendisse gravida purus risus, id aliquet elit vulputate et. Vivamus ut dignissim lacus. Suspendisse at pulvinar justo, et sagittis erat. Praesent tincidunt urna lorem, vitae euismod lectus pulvinar ut. Morbi tincidunt porta sem, et vestibulum nunc congue eget. Donec vitae tortor vitae leo pretium pretium. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Integer dictum porttitor suscipit. Ut blandit, ex vel ultricies tempus, elit urna fermentum arcu, ac suscipit mi dui ut dui. Ut rhoncus justo quis justo tempus pulvinar. Proin malesuada blandit ante, a pulvinar eros vulputate ac. Cras ac ornare lectus.</p>
-        </div>
-      </div>
-    </Container>
-  </div>
+const bigBro = require('../../assets/thibaudbonnevial.jpg')
+const littleBro = require('../../assets/noebonnevial.jpg')
+const smith = require('../../assets/smith.jpg')
 
-export default Equipe
\ No newline at end of file
+const Team = () => (
+  <Container>
+    <h2><FormattedMessage id='equipe.title' /></h2>
+    <Card.Group centered items>
+      <Card>
+        <Card.Content>
+          <Image src={bigBro} alt='' />
+          <Card.Header>Thibaud Bonnevial</Card.Header>
+          <Card.Meta>Noe's Bro</Card.Meta>
+          <Card.Description>Thibaud is a nice guy with long hair</Card.Description>
+        </Card.Content>
+        <Card.Content extra>
+          <p>Tree killer</p>
+        </Card.Content>
+      </Card>
+      <Card>
+        <Card.Content>
+          <Image src={littleBro} alt='' />
+          <Card.Header>Noe Bonnevial</Card.Header>
+          <Card.Meta>Thibaud's Bro</Card.Meta>
+          <Card.Description>Noe is a nice guy with short hair</Card.Description>
+        </Card.Content>
+        <Card.Content extra>
+          <p>Tree healer</p>
+        </Card.Content>
+      </Card>
+      <Card>
+        <Card.Content>
+          <Image src={smith} alt='' />
+          <Card.Header>Smith Stan</Card.Header>
+          <Card.Meta>Bonnevial's employee</Card.Meta>
+          <Card.Description>Smith is a patient man working with Bonnevial's bros</Card.Description>
+        </Card.Content>
+        <Card.Content extra>
+          <p>Tree lover</p>
```

Comment:
Tous les _string_ comme ça doivent aller dans les fichiers de intl _fr | en |....json_
Puis tu dois utiliser le component `<FormattedMessage id='message_id' />`. Par exemple :
```suggestion
          <p><FormattedMessage id='team.tree.description' /></p>
```